index.html

```diff
@@ -9,88 +9,103 @@
 		<script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script>
 	<![endif]-->
 </head>
-
 <body>
+	<div class="container">
+<!-- Menu Content  -->
+		<header>
+		S&J
+			<nav>
+			Home
```

Comment:
remember to make a choice about how you will represent these. they might be a tags or an unordered list. the choice is yours